### PR TITLE
Corrected typos and instructions in the Quick Start and Base Tutorials.

### PR DESCRIPTION
**Pull request by jconlon**
Here are the changes to the website.

### CODE REVIEW

**Comment by bjhargrave** [comment]
@jconlon Please amend your commit message to add the necessary `Signed-off-by` as requested by <https://github.com/osgi/osgi.enroute.site/blob/gh-pages/CONTRIBUTING.md#sign-your-work>. Thanks.

File: _tutorial_base/800-ci.md [hunk 1]
```diff
@@ -18,13 +18,13 @@ So we only have to fire up a shell and run the build!
 
 You do not even need to have gradle installed. There is a script included, `gradlew`, that will download the correct gradle version. You can also install gradle (>= 1.12) and use it. It should go without saying, but you should also have java 8 installed on the command line.
 
-	$ cd /Ws/com.acme.prime
 	$ java -version
 	java version "1.8.0"
+	$ cd /Users/aqute/git/com.acme.prime
```
Comment (bjhargrave) [inline _tutorial_base/800-ci.md]: This is right in the middle of the output of `java -version`.

Comment (jconlon) [inline _tutorial_base/800-ci.md]: Just changed what Peter had.  Guess that what the dots ... mean.  I just wanted to change it from the Eclipse workspace to the Git/BND workspace.

Comment (jconlon) [inline _tutorial_base/800-ci.md]: Amended my last comment on my local repo but having difficulty doing a
git push --force origin master 
to gitHub.  Any suggests or can I just add another commit noting that previous commits are signed off?

Comment (bjhargrave) [inline _tutorial_base/800-ci.md]: origin is probably the main osgi.enroute.sire repo rather than your fork. You need to push to your fork.

Comment (jconlon) [inline _tutorial_base/800-ci.md]: Done

Comment (bjhargrave) [inline _tutorial_base/800-ci.md]: Thanks for fixing the commit comment. But you still did not fix the fact that you moved the `cd` command from the first command in the example to in the middle of the output from the `java -verison` command. 

It used to look like this:

    $ cd /Ws/com.acme.prime
    $ java -version
    java version "1.8.0"
    Java(TM) SE Runtime Environment ...
    $ ./gradlew

but now looks like this:

    $ java -version
    java version "1.8.0"
    $ cd /Users/aqute/git/com.acme.prime
    Java(TM) SE Runtime Environment ...
    $ ./gradlew

See <https://github.com/jconlon/osgi.enroute.site/blob/8060a439d5a472606fcdfe8ec3973da9cb95b0a4/_tutorial_base/800-ci.md#building-outside-the-ide>

Comment (jconlon) [inline _tutorial_base/800-ci.md]: Got it.

Comment (jconlon) [inline _tutorial_base/800-ci.md]: There are more typos in this page.  I am not sure how to proceed with it as the gradle task shown on the page: 

./gradlew export.com.acme.prime.eval

is failing when calling the task from the base of the workspace.  
Is this a bug, or is there something I am missing?

**Comment by pkriens** [comment]
Thanks, I really need this kind of feedback at the moment.

**Comment by jconlon** [comment]
Peter - Its the least I can do.

Note that on our Continuous Integration 
<http://enroute.osgi.org/tutorial_base/800-ci.html> page there is an 
outstanding issue ongoing with the gradle export.xxx task.

BndTools Forum Topic 
<https://groups.google.com/forum/#%21topic/bndtools-users/MPFljUPNGUU>

BndFix#798 <https://github.com/bndtools/bnd/pull/798>

-- 
{ name     : "John Conlon",
   title    : "Principal Consultant",
   phone    : "(608)620-4938 x100",
   cell     : "(608)620-8867",
   email    : "jconlon@verticon.com",
   web   : ["http://www.verticon.com","http://tracker.verticon.com"]}

**Comment by jconlon** [comment]
Pushing forward with review of the Principles page, but encountered a 
paragraph that I need some assistance with:

> It is interesting to see how we lack proper terminology in our 
> industry. In maven <http://maven.apache.org/> we talk about an 
> artifact but it is not clear if it refers to the bits on disk (the JAR 
> file), or the project that builds it, or something maybe even 
> something else? In this document we use /program/ for the what is the 
> combination of /groupId/ and /artifactId/ in maven and /revision/ when 
> a specific /version/ is added. The term /project/ defines the concept 
> of a set of programs that can be used to /build/ a revision.

Here is my initial rework:

It is interesting to see how we lack proper terminology in our industry. 
In maven <http://maven.apache.org/> we talk about an artifact but it is 
not clear if it refers to the bits on disk (the JAR file), or the 
project that builds it, or maybe even something else?  In this document 
we define an artifact as the combination of maven's groupId and 
artifactId and a revision when a specific version is added.  The term 
project defines the concept of a set of artifacts that can be used to 
build a revision.

Does that communicate what was intended?

John

**Comment by bjhargrave** [comment]
@jconlon You should create a new PR or issue for additional changes. This PR is already merged and closed.

**Comment by pkriens** [comment]
Don't spend too much time on the principles. Needs a good scrub from me.

**Comment by jconlon** [comment]
Yes, I agree.  Needs a good 'wordsmithing'.   Great ideas are difficult 
to communicate to people, especially to those who hold misconceptions.

Nevertheless, time spent was worth it to grok some of the more subtle 
points.   Big take away was your counter to the 'fidelity' idea and the 
clarification of the need for build and runtime dependency control.

**Pull request merged**